includes/blocks.inc

```diff
@@ -422,7 +422,7 @@ function islandora_solr_simple_search_form($form, &$form_state) {
   $form['simple']["islandora_simple_search_query"] = array(
     '#size' => '15',
     '#type' => 'textfield',
-    '#title' => '',
+    '#title' => 'Search For:',
```

Comment:
A couple points here:
* The string should be inside a [`t()`](https://api.drupal.org/api/drupal/includes!bootstrap.inc/function/t/7.x) call, to expose it to Drupal's translation mechanism
* The colon should probably be left off, as it is more of a theme element... Also, may be taken in the spirit of a [heading or page title](https://www.drupal.org/drupalorg/style-guide/content#headings), in which you should "Never end [...] with a [...] colon, [...] etc."

Also, if this is just to fulfill requirements for screenreaders (and isn't necessary to display for all users), should we be adding [`#title_display`](https://api.drupal.org/api/drupal/developer!topics!forms_api_reference.html/7.x#title_display) to be `invisible` (or possibly configurable in the settings for the block, and defaulting to `invisible`?)?

Also, speaking opinion here, but would just 'Search' suffice? 'Search For' may be clumsier to translate (and take up additional screen real estate if/when shown), but I'm not sure.